### ISSUE TITLE
Can't import contacts from google contacts app (synchronization is disabled)

### ISSUE THREAD

**Issue by Solarunit** [SimpleMobileTools/Simple-Contacts#372]
I can't import contacts from google contacts app. Synchronization with google is disabled, so it is device-only contacts.

**Comment by tibbi** [comment]
not sure what you mean. You see some contacts in the Google Contacts app, but you cannot see them in Simple Contacts? Device-only contacts should still be visible in both apps.

**Comment by Solarunit** [comment]
> You see some contacts in the Google Contacts app, but you cannot see them in Simple Contacts?

Yes. I've just created 2 contacts: _contact_1_ (in Google Contacts) and _contact_2_ in Simple Contacts. And I don't see _contact_1_   in Simple Contacts and vice versa.
Also import contacts from .vcf file (created via Google Contacts) doesn't work.

**Comment by tibbi** [comment]
do you see any of the contacts in some third Contacts app? You could download one for testing. Also are you getting some error at importing .vcf file, or what do you mean by not working?

**Comment by Solarunit** [comment]
> do you see any of the contacts in some third Contacts app? 

 I can see all contacts via Google Phone app (contacts tab)

> Also are you getting some error at importing .vcf file, or what do you mean by not working?

No error. Nothing happens. I have tried both options - Target contacts source:

- Google account
- Phone storage (not visible by other apps)

**Comment by tibbi** [comment]
if you open the .vcf file with any text editor, does it look somewhat like this?
BEGIN:VCARD
VERSION:2.1
N:Name;;;;
TEL;CELL:123-456
END:VCARD

**Comment by Solarunit** [comment]
BEGIN:VCARD
VERSION:2.1
N;CHARSET=UTF-8;ECODING=QUOTED-PRINTABLE:;=7H=TT=7U..........
FN;CHARSET=UTF-8;ECODING=QUOTED-PRINTABLE:;=7H=TT=7U..........
TEL;CELL:123-456
END:VCARD

**Comment by tibbi** [comment]
oh gosh, Ill try creating a sample .vcf from Google Contacts and handle it properly. If I remember correctly vcf created via the website was simpler. I guess you have some special characters in the contact names, like diacritics? 
Not sure about the initial issue, why arent contacts visible in the app in the first place, will try reproducing it someday.

**Comment by Solarunit** [comment]
> I guess you have some special characters in the contact names, like diacritics?

I don't use special characters.
I often fill the name field as _John Smith (Company name)_
